src/main/java/org/s1ck/gdl/GDLLoader.java

```diff
@@ -304,6 +308,12 @@ public void enterGraph(GDLParser.GraphContext graphContext) {
     currentGraphId = g.getId();
   }
 
+  private boolean isEmpty(GDLParser.GraphContext ctx) {
```

Comment:
Maybe we can just have `isEmpty(HeaderContext header, PropertiesContext  properties)` and use that function in all three places?